### PR TITLE
Fix SanitizerConfig type definition

### PR DESCRIPTION
**Pull request by hata6502**
https://github.com/codex-team/editor.js/issues/1513

### CODE REVIEW

File: src/types-internal/html-janitor.d.ts [hunk 1]
```diff
@@ -3,9 +3,11 @@
  * After that we can use it at the TS modules
  */
 declare module 'html-janitor' {
+  type Option = boolean | { [attr: string]: boolean | string };
+
   interface Config {
     tags: {
-      [key: string]: boolean|{[attr: string]: boolean|string}|(() => any)
+      [key: string]: Option | ((el: HTMLElement) => Option)
```
Comment (hata6502) [inline src/types-internal/html-janitor.d.ts:10]: By adding `el: HTMLElement`, escape Type error with callback.

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,7 @@
 - `Fix` — Fix problem with entering to Editor.js by Tab key [#1393](https://github.com/codex-team/editor.js/issues/1393)
 - `Fix` - Sanitize pasted block data [#1396](https://github.com/codex-team/editor.js/issues/1396).
 - `Fix` - Unnecessary block creation after arrow navigation at last non-default block[#1414](https://github.com/codex-team/editor.js/issues/1414)
+- `Fix` - Fix SanitizerConfig type definition[#1456](https://github.com/codex-team/editor.js/pull/1456)
```
Comment (neSpecc) [inline docs/CHANGELOG.md:13]: pls, move this line to the 2.19.2 section 
```suggestion
- `Fix` - Fix SanitizerConfig type definition [#1456](https://github.com/codex-team/editor.js/pull/1456)
```

File: src/types-internal/html-janitor.d.ts [hunk 3]
```diff
@@ -3,9 +3,11 @@
  * After that we can use it at the TS modules
  */
 declare module 'html-janitor' {
+  type Option = boolean | { [attr: string]: boolean | string };
```
Comment (neSpecc) [inline src/types-internal/html-janitor.d.ts:6]: docs missed

File: types/configs/sanitizer-config.d.ts [hunk 4]
```diff
@@ -1,3 +1,5 @@
+type Option = boolean | { [attr: string]: boolean | string };
```
Comment (neSpecc) [inline types/configs/sanitizer-config.d.ts:1]: docs are missed

**Review by neSpecc — CHANGES_REQUESTED** [review]
PR description is not provided

**Comment by hata6502** [comment]
@neSpecc 

Sorry, I sent a issue. 
https://github.com/codex-team/editor.js/issues/1513

I feel great to install Contribution Inspector!
https://github.com/codex-team/editor.js/pull/1512

**Comment by hata6502** [comment]
@neSpecc
Would you review or edit this PR?

**Review by khaydarov — APPROVED** [review]

**Comment by neSpecc** [comment]
Please pull the `next` branch

**Comment by khaydarov** [comment]
@neSpecc I did

File: src/components/modules/blockManager.ts [hunk 5]
```diff
@@ -560,7 +560,7 @@ export default class BlockManager extends Module {
    * 2) Mark it as current
    *
    * @param {Node} childNode - look ahead from this node.
-   * @returns can return undefined in case when the passed child note is not a part of the current editor instance
+   * @returns {Block | undefined} can return undefined in case when the passed child note is not a part of the current editor instance
```
Comment (hata6502) [inline src/components/modules/blockManager.ts:563]: Fixed  for `yarn lint:fix`

**Comment by hata6502** [comment]
Resolved conflicts.

**Review by gohabereg — APPROVED** [review]

**Review by neSpecc — APPROVED** [review]

**Pull request merged**